### DISCUSSION TITLE
Rejected even though I am a student and can't reapply.

### DISCUSSION THREAD

**Discussion by connorjohnson1** [community/community#139992] [GitHub Education]
### Hi everyone,

I am a PhD student starting my analysis. I tried to apply for Github education and was denied for no obvious reason. I sent my ID but it criticized that I didn't use device camera. I'm on a desktop?? And I am in a classroom from  over a year ago and it is asking me to make more contributions before I reapply... I am not in this class anymore. I don't know if I should leave it or if that would make things work. It seems Github is perfectly happy denying students and not offering any solutions. Has anyone been in this situation and made it out??

I am transitioning to python from Matlab and would really like to use co-pilot.

**Comment by Sanidhyafeaturist** [comment]
Many students have encountered similar issues, especially with ID verification. If the application requires a device camera, using a smartphone to submit your ID could be a workaround. Regarding the contribution requirement, you might consider making a few small contributions to public repositories to meet that threshold before reapplying. If you're still facing challenges, reaching out directly to GitHub support might provide clarity.
